pkg/controller/migmigration/task.go

```diff
@@ -316,21 +318,6 @@ func (t *Task) Run() error {
 				return liberr.Wrap(err)
 			}
 		}
-
-	case Prepare:
```

Comment:
Deletion of stage pods and annotation cleanup have are still run in the same spot during a migration, but now they are individual phases instead of being bundled together in `Prepare`